addons/web/static/tests/fields/relational_fields/field_one2many_tests.js

```diff
@@ -3438,7 +3437,7 @@ QUnit.module('fields', {}, function () {
             assert.expect(1);
 
             this.data.partner.fields.p.default = [
-                [0, false, { date: '2017-10-08' }],
+                [0, false, { date: '2017-10-08', p: [] }],
```

Comment:
which difference between not providing p, and setting it to the empty list?

Reply:
Infinite recursion. It's a model having an auto create relation to itself.

Reply:
Providing p as the empty command (or empty recordset) makes it stop using the default value (which is to keep creating new records).

Reply:
ok